src/features/defi/providers/foxy/components/FoxyManager/Withdraw/components/Withdraw.tsx

```diff
@@ -89,6 +96,13 @@ export const Withdraw: React.FC<StepComponentProps> = ({ onNext }) => {
     [asset.precision, cryptoAmountAvailable, marketData.price, setValue],
   )
 
+  const accountFilter = useMemo(() => ({ accountId: accountId ?? '' }), [accountId])
+  const accountMetaData = useAppSelector(state =>
+    selectPortfolioAccountMetadataByAccountId(state, accountFilter),
+  )
+
+  const bip44Params = accountMetaData?.bip44Params
+
   const handleContinue = useCallback(
     async (formValues: FoxyWithdrawValues) => {
       if (!state?.userAddress || !dispatch || !rewardId || !api) return
```

Comment:
|| !bip44Params

Reply:
Nice catch, here we go 44ec7832650557975cc6afd98d8ff1f04fca744b